components/ui/customtable/footer/TableFooter.js

```diff
@@ -23,20 +23,22 @@ class TableFooter extends PureComponent {
 
   render() {
     const { pagination, showTotalPages } = this.props;
-
-    return (
-      <div className="table-footer">
-        <Paginator
-          options={pagination}
-          onChange={page => this.onChangePage(page)}
-        />
-
-        {(pagination.enabled && showTotalPages && pagination.pages) &&
-          <div>Page <span>{pagination.page}</span> of <span>{pagination.pages}</span></div>
-        }
-
-      </div>
-    );
+    if (pagination.size > 0) {
+      return (
+        <div className="table-footer">
+          <Paginator
+            options={pagination}
+            onChange={page => this.onChangePage(page)}
+          />
+
+          {(pagination.enabled && showTotalPages && pagination.pages) &&
+            <div>Page <span>{pagination.page}</span> of <span>{pagination.pages}</span></div>
+          }
+
+        </div>
+      );
+    }
+    return null;
```

Comment:
We haven't had good experiences with this thing. Depending on where renders it can even crash the application. Let's try to avoid it.